mitiq/observable/pauli.py

```diff
@@ -84,7 +84,7 @@ def __init__(
 
     @property
     def coeff(self) -> complex:
-        return self._pauli.coefficient
+        return self._pauli.coefficient  # type: ignore
```

Comment:
Why is this needed now?

Reply:
I guess `cirq.PauliString` now supports symbolic coefficients beyond complex coefficients. In Mitiq instead we only support /assume complex coefficients. `mypy` don't like this fact.